python/paddle/fluid/dygraph/learning_rate_scheduler.py

```diff
@@ -270,12 +270,12 @@ def __init__(
         self.staircase = staircase
 
     def step(self):
-        from .. import layers
+        import paddle
```

Comment:
why this? this may cause performance issue

Reply:
Done, already remove the code in the file.